lib/taskana-core/src/test/java/acceptance/task/delete/DeleteTaskAccTest.java

```diff
@@ -29,6 +29,15 @@ class DeleteTaskAccTest extends AbstractAccTest {
   @WithAccessId(user = "user-1-2")
   @Test
   void testDeleteSingleTaskNotAuthorized() {
+    ThrowingCallable call =
+        () -> taskService.deleteTask("TKI:000000000000000000000000000000000037");
+
+    assertThatThrownBy(call).isInstanceOf(NotAuthorizedException.class);
+  }
+
+  @WithAccessId(user = "user-taskrouter")
```

Comment:
"UserIsMember" -> here is no group used, is it?

Reply:
"Memer of role" should mean that the user has the role. No group is used. Should I rename the test to "_When_UserHasTaskRouterRole"?

Reply:
again, my fault. Sorry. I stopped reading at 'member'... :-) Fine for me the way it is.